Volume_1/Book_about_Quadratization.log

```diff
@@ -1,542 +1,488 @@
-This is pdfTeX, Version 3.14159265-2.6-1.40.18 (TeX Live 2017/Debian) (preloaded format=pdflatex 2022.3.21)  7 APR 2022 20:26
```

Comment:
Can you show me where I asked for a commit  to update the `.log` file? I asked for the `.pdf` and the `.tex` files to be modified.

Reply:
I was looking at some previous pull requests to figure out how to compile the LaTeX file into a PDF, and in one of them you wanted a `.log` file, which I got confused with the instructions for my pull request.

Reply:
Okay. I think that previous discussion, was the reason why I asked you only to commit the `.pdf` and the `.tex` files. If you were confused it would have been a good idea to ask before pushing a commit. Anyway, what's done is done.